libs/core/news/helpers/normalize-contentful-data/normalize-contentful-data.helper.ts

```diff
@@ -3,47 +3,48 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-import { Asset, Entry, EntryCollection } from "contentful"
+import { EntryCollection } from "contentful"
 import { NewsEntry } from "../../dto"
 import { getBase64 } from "../get-base-64/get-base64.helper"
+import { getAssetForEntry } from "./get-asset-for-entry.helper"
+import {
+  CARD_IMAGE_MAX_HEIGHT_PIXEL,
+  CARD_IMAGE_MAX_WIDTH_PIXEL,
+} from "../../../news/components/card/card.constans"
+
+const DPI = 2
+const WIDTH = CARD_IMAGE_MAX_WIDTH_PIXEL * DPI
+const HEIGHT = CARD_IMAGE_MAX_HEIGHT_PIXEL * DPI
+const QUALITY = 100
 
-// AUTO DISABLED - fix me if you like :)
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export const normalizeContentfulData = async (
   data: EntryCollection<NewsEntry>
-) => {
-  // AUTO DISABLED - fix me if you like :)
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+): Promise<{ newsItems: NewsEntry[]; lastUpdate: string }> => {
   const { items, includes } = data
-  const news = items.map(({ fields, sys }: Entry<NewsEntry>) => {
-    return {
+  const newsItems: NewsEntry[] = []
+
+  for (const item of items) {
+    const { fields, sys } = item
+    const { title, url: relativeUrl } = getAssetForEntry(
+      includes.Asset,
+      fields?.image?.sys?.id
```

Comment:
'sys.id' field is always available for contentful assets. Are you sure that you want to use 'image.sys.id' instead?